presidio-cli/.presidiocli

```diff
@@ -0,0 +1,17 @@
+entities:
+  - CREDIT_CARD
+  - CRYPTO
+  - DATE_TIME
+  - DOMAIN_NAME
+  - EMAIL_ADDRESS
+  - IBAN_CODE
+  - IP_ADDRESS
+  - NRP
+  - LOCATION
+  - PERSON
+  - PHONE_NUMBER
+  - MEDICAL_LICENSE
+ignore: |
+  .git
+  .pytest_cache
+  .vscode
```

Comment:
If `language` is also a field in this file, let's add it for others to change if needed.

Reply:
Added `language` field here as well.